obspy/core/tests/test_util_types.py

```diff
@@ -1,14 +1,17 @@
 # -*- coding: utf-8 -*-
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from future.builtins import *  # NOQA
-
```

Comment:
Please keep order of imports.. future.builtins must come right after the `__future__` import

Reply:
This still needs to be done.